MC-10287: Added documentation of the product catalog and pricing (#141)
cloudops/cloudmc-api-docs: source/includes/administration/_pricings.md

### Pricings

> The pricings determines the price for each product defines for a selected catalogs.

Reviewer: ```suggestion
The pricing determines the price for each product defined for a selected catalog.
```

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

          "unitPrice": 123.0,
          "product": {
            "metricType": "GAUGE",
            "unit": {
              "unit": "HOUR",
              "name": {}
            },
            "period": "HOUR",
            "name": {
              "en": "vCPU",
              "fr": "vCPU"
            },
            "transformer": {
              "type": "PROPORTIONAL_TO_TIME"
            },
            "id": "342f0d9b-3c3c-456d-b853-31d713b79e72",
            "attribute": "RAWUSAGE",
            "source": "1",
            "filters": [],
            "sku": "vCPU",
            "categoryId": "fcd8908d-e1a0-41f2-9c38-fed1d1f77f18"
          },
          "cogs": 321.0,
          "currency": "CAD",
          "id": "45425fe6-beb3-4c78-be68-f82eeb3976c6"
        }
      ],
      "name": {
        "en": "cloud.ca default pricing",
        "fr": "prix par défaut cloud.ca"
      },
      "description": {
        "en": "cloud.ca default pricing",
        "fr": "prix par défaut cloud.ca"
      },
      "id": "096bbfee-a6e6-4373-8a4e-1a8b0f2bf484",
      "effectiveDate": "2020-05-08T16:53:45Z"
    }
  ]
}
```

Attributes | &nbsp;
---- | -----------
`id`<br/>*UUID* | The UUID of the pricing.

> `name`<br/>*Object* | The name object in each language for the pricing (en, fr or es).

Reviewer: ```suggestion
`name`<br/>*Object* | The name object in each language for the pricing (en, fr, or es).
```

Reviewer 2: We probably shouldn't hardcode the languages because they vary per branding.

Author: How should i describe it then? Just removing the parenthesis seems not complete

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

> `description`<br/>*string* | The description object in each language for the pricing (en, fr or es).

Reviewer: ```suggestion
`description`<br/>*string* | The description object in each language for the pricing (en, fr, or es).
```

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

> `supportedCurrencies`<br/>*Array[string]* | List of the currency which are link to this pricing.

Reviewer: ```suggestion
`supportedCurrencies`<br/>*Array[string]* | List of the currencies which are supported by this pricing.
```

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

> `productCatalogs`<br/>*Object* | List of product catalog object which are attached to this pricing.

Reviewer: ```suggestion
`productCatalogs`<br/>*Object* | List of product catalog objects which are attached to this pricing.
```

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

`organization.id`<br/>*UUID* | UUID of the organisation to which belongs the pricing.

> `pricingProducts`<br/>*Array[Object]* | The list of products assigned to the pricing.

Reviewer: ```suggestion
`pricingProducts`<br/>*Array[Object]* | The list of product pricings assigned to the pricing.
```
? would this maybe be more clear

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

            "sku": "vCPU",
            "categoryId": "fcd8908d-e1a0-41f2-9c38-fed1d1f77f18"
          }
        ]
      }
    ],
    "organization": {
      "id": "409c1162-a930-4207-bdf9-710580c3542b"
    },
    "pricingProducts": [
      {
        "unitPrice": 123.0,
        "product": {
          "metricType": "GAUGE",
          "unit": {
            "unit": "HOUR",
            "name": {}
          },
          "period": "HOUR",
          "name": {
            "en": "sfg",
            "fr": "sdfg"
          },
          "transformer": {
            "type": "PROPORTIONAL_TO_TIME"
          },
          "id": "342f0d9b-3c3c-456d-b853-31d713b79e72",
          "attribute": "RAWUSAGE",
          "source": "1",
          "filters": [],
          "sku": "sdfg",
          "categoryId": "fcd8908d-e1a0-41f2-9c38-fed1d1f77f18"
        },
        "cogs": 321.0,
        "currency": "USD",
        "id": "45425fe6-beb3-4c78-be68-f82eeb3976c6"
      }
    ],
    "name": {
      "en": "testen",
      "fr": "testfr"
    },
    "description": {
      "en": "descen",
      "fr": "testfr"
    },
    "id": "096bbfee-a6e6-4373-8a4e-1a8b0f2bf484",
    "effectiveDate": "2020-05-08T16:53:45Z"
  }
}
```

> Attributes | &nbsp;

Reviewer: the same comments as above

Author: f428ed5b76ae49b8bf4e644a9c5cd49c51399110
